index.html

```diff
@@ -116,17 +115,12 @@ <h2 class="tonic">Tonic</h2>
 
    
    <div class="section4">
-    <row>
-      <col>
-    <img src="images/d.png" class="imagesec4"></div>
-    </col>
-
     
-    <col>
+    <img src="images/d.png" alt="ss" class="imagesec4"></div>

```

Comment:
- [ ] [REQUIRED] I can see that you are using the "root path" method for your the `src` attribute of the `<img>` tags. While this should work perfectly on your machine. On servers, it doesn't always work and that would result in broken images as shown in the screenshot below. You can use the "dot path" way instead (for further explaination see the example below)

## screenshot

> a screenshot of broken images

![image](https://user-images.githubusercontent.com/60979926/178114137-a6271f18-d501-4254-9991-7f1e1dd7314a.png)

## example

### before

```html
<img src="pic/icon/img.png" />
```

### after

```html
<img src="./pic/icon/img.png" />
```

Reply:
- [ ] Kindly implement the changes requested by the previous reviewer.